docs/kolaGuide.md

```diff
@@ -0,0 +1,80 @@
+ # Kola Introduction Documentation
+ 
+This document goes through the basics of how kola tests work and how someone just beginning can get started
+ 
+### What is Kola?
+Kola is a framework for testing software integration in CoreOS systems across multiple platforms. It is primarily designed to operate within the CoreOS Assembler for testing software that has landed in the OS image.
+
+
+> [Additional documentation about Kola](https://github.com/coreos/coreos-assembler/tree/main/docs/kola)
+ 
+ ### Steps to run a test from start to finish
+ 
+ 1. `cosa run` (Creates and brings you into the shell of a QEMU instance, running the provided version of FCOS or RHCOS)
+ 2. `cosa kola list` (Here we can see all of the tests that we can do)
+ 3. `cosa kola run <test-pattern>` (This will run a specific test and check to see if a test passes)
+ 4. `cosa kola run ext.config*` (This will run all the tests that start with `ext.config`)
+
+### More information on tests
+
+There are file logs in each of these locations: They will help you to debug the problem and will certainly give you hints along the way
+
+1. `journal.txt`
+2. `console.txt` 
+3. `ignition.json` 
+4. `gunzip journal-raw.txt.gz` 
+
+
+
+### Extended artifacts
+
+1. Extended artefacts need additional forms of testing (You can pass the ignition and the path to the artefact you want to test)
+2. `cosa kola run -h` (this allows you to see the commands yourself and what syntax is needed)
+2. `cosa buildextend-"name_of_artifact` (An example of building an artefact before testing it) 
+3. `kola run -p <platform>` Is the most generic way of testing extended artifacts, this is mostly useful for the cloud platforms
+4. For running the likes of metal/metal4k artifacts there's not much difference than running `kola run` from the coreos-assembler
+5. `cd builds/latest/` (This will show your latest build information)
+6. In the case of the `testiso` command, you'll see that there is the `--qemu-native-4k` option passed to `kola testiso`.  This instructs the `testiso` test to attempt to install FCOS/RHCOS to a disk that uses 4k sector size.  If you don't include that option, the `testiso` command will attempt to install FCOS/RHCOS to a non 4k disk (512b sector size)
+7. `kola testiso -S --scenarios pxe-install,pxe-offline-install --output-dir tmp/kola-metal` 
```

Comment:
```suggestion
7. `kola testiso -S --scenarios pxe-install,pxe-offline-install --output-dir tmp/kola-metal`   You can also pass some scenarios, in case you do not want to run all of them